CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
+## 8.6.3 - March 5, 2020
+[Changes](https://github.com/mapbox/mapbox-gl-native-android/compare/android-v8.6.2...android-v8.6.3) since [Mapbox Maps SDK for Android v8.6.2](https://github.com/mapbox/mapbox-gl-native-android/releases/tag/android-v8.6.2):
+### Bug fixes
+ - Keeps telemetry events not being obfuscated [#215](https://github.com/mapbox/mapbox-gl-native-android/pull/215)
```

Comment:
@Chaoba Let's use the same wording from the 9.1.0 changelog, and link to the original PR rather than the cherrypick:

> Fix Proguard configuration to prevent obfuscation of telemetry event classes. #195

(I'm assuming that the cherrypick is the same as #195; the PR you linked to here #215 is a closed with no info.)